packages/fluentui/react-northstar/src/themes/teams-dark-v2/siteVariables.ts

```diff
@@ -148,3 +148,9 @@ export const colorScheme = {
     backgroundDisabled1: colors.grey['800'],
   },
 };
+
+// SHADOW LEVELS
+export const shadowLevel1 = '0 0 2px rgba(0,0,0,0.30), 0 2px 4px rgba(0,0,0,0.25)';
```

Comment:
nothing is using these shadows, is that intentional? are updates planned in a future PR?